src/js/views/maps/ExpansionPanelView.js

```diff
@@ -70,16 +71,30 @@ define(
          * @property {boolean} startOpen Whether the panel should be expanded by
          * default. 
          */
-        initialize({ title, contentViewInstance, icon, panelsModel, startOpen }) {
-          this.templateVars.title = title;
-          this.templateVars.icon = icon;
+        initialize({ title, contentViewInstance, icon, panelsModel, startOpen, isSvgIcon, }) {
+          this.templateVars = {
+            ...this.templateVars,
```

Comment:
```suggestion
            classNames: CLASS_NAMES,
```

and then you could delete lines 53-58:
```js
        /** Values meant to be used by the rendered HTML template. */
        templateVars: {
          classNames: CLASS_NAMES,
          title: '',
          icon: '',
        },
```